.github/workflows/api.yml

```diff
@@ -3,9 +3,15 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'packages/api/**'
+      - '.github/workflows/api.yml'
   pull_request:
```

Comment:
Should we just remove this part and the `if` condition below?

Reply:
No, we want the test jon to run for pull requests that affect the api code. I could split these out into MOAR yamlz, if you'd prefer.

Reply:
Ah, no, no more yamls - I missed the existing test job for some reason 🤦 .